sql/core/src/main/scala/org/apache/spark/sql/execution/DataSourceScanExec.scala

```diff
@@ -325,8 +325,7 @@ case class FileSourceScanExec(
   }
 
   @transient
-  private val pushedDownFilters = dataFilters.flatMap(DataSourceStrategy.translateFilter)
-  logInfo(s"Pushed Filters: ${pushedDownFilters.mkString(",")}")
```

Comment:
It is confused the log will print twice.

Reply:
If the log is consistent, I think it should not be confused.

Reply:
@ulysses-you . Please don't piggy-back independent item into your PR. This is totally irrelevant to your PR (`Make FileSourceScanExec class fields lazy`)

Reply:
Yes, it is a related change.

Reply:
Got it.